### PR TITLE
UX-287 Limit Modal Content Height

### PR DESCRIPTION
**Pull request by logansparlin**
### What Changed

- Limit modal content height and overflow scroll

### How To Test or Verify

- Verify Modal stories in Storybook

### PR Checklist

- [ ] Add the correct `type` label
- [ ] Pull request approval from #uxfe or #design-guild

### CODE REVIEW

File: packages/matchbox/src/components/Modal/Content.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const Content = React.forwardRef(function Content(props, ref) {
   const { children } = props;
 
   return (
-    <Box data-id="modal-content" p="500" ref={ref}>
+    <Box data-id="modal-content" p="500" maxHeight="60vh" overflow="scroll" ref={ref}>
```
Comment (jonambas) [inline packages/matchbox/src/components/Modal/Content.js:10]: I would change this to overflowY so the horizontal scrollbar are hidden if someone has them enabled

Screen Shot 2020-08-12 at 9 02 57 AM

File: packages/matchbox/src/components/Modal/Modal.js [hunk 2]
```diff
@@ -94,7 +94,13 @@ const Modal = React.forwardRef(function Modal(props, ref) {
             role="dialog"
             aria-modal="true"
           >
-            <Box p={['400', null, '700']} size="100%" ref={overlayRef} data-id="modal-overlay">
+            <Box
+              p={['400', null, '700']}
+              pt={['5rem', null]}
```
Comment (jonambas) [inline packages/matchbox/src/components/Modal/Modal.js:99]: should we use a token?

Comment (logansparlin) [reply packages/matchbox/src/components/Modal/Modal.js:99]: Might have to create a new one. Our tokens jump from 4rem to 10rem and 4rem is too small and the modal is still hidden behind the app header, 10rem is pretty large but I guess we could use it. What do you think?

Comment (jonambas) [reply packages/matchbox/src/components/Modal/Modal.js:99]: Oh yea, i'd just leave it at 5rem then, doesn't make sense to make an `850` token at `5rem`

**Pull request merged**